src/CaptureClient/ApiEventProcessor.cpp

```diff
@@ -128,4 +133,203 @@ void ApiEventProcessor::ProcessTrackingEvent(const orbit_api::ApiEvent& api_even
   capture_listener_->OnTimer(timer_info);
 }
 
+namespace {
+void EncodedColorToColor(uint32_t encoded_color, orbit_client_protos::Color* color) {
+  color->set_red((0xff000000 & encoded_color) >> 24);
+  color->set_green((0x00ff0000 & encoded_color) >> 16);
+  color->set_blue((0x0000ff00 & encoded_color) >> 8);
+  color->set_alpha(0x000000ff & encoded_color);
```

Comment:
nit: I would move flags on the right and do something like this `((encoded_color & (0xff << 24)) >> 24)`

Reply:
I kind of feel like it's more readable as it is, just my feeling.

Reply:
I kind of like @dimitry-'s suggestion a bit more, so I went with this.